pupil-spa/src/app/instructions/instructions.component.html

```diff
@@ -5,8 +5,8 @@ <h1 class="heading-xlarge">Instructions</h1>
 <div>
   <ul class="text list list-bullet">
     <li>This check has <strong>{{count}} questions</strong>.</li>
-    <li>Each question will show for <strong>5 seconds</strong>.</li>
-    <li>There will be a <strong>2-second</strong> pause between each question.</li>
+    <li>Each question will show for <strong>{{questionTime}} seconds</strong>.</li>
+    <li>There will be a <strong>{{loadingTime}}-second</strong> pause between each question.</li>
```

Comment:
Do we need the '-', as now its reads like:
"There will be a 3-second pause between each question."
@jon-shipley